thorlcr/activities/thdiskbase.cpp

```diff
@@ -68,20 +68,19 @@ void CDiskReadMasterBase::init()
             mapping.setown(getFileSlaveMaps(file->queryLogicalName(), *fileDesc, container.queryJob().queryUserDescriptor(), container.queryJob().querySlaveGroup(), local, false, hash, file->querySuperFile()));
             addReadFile(file, temp);
         }
-        IDistributedSuperFile *super = file->querySuperFile();
-        unsigned numsubs = super?super->numSubFiles(true):0;
         if (0 != (helper->getFlags() & TDRfilenamecallback)) // only get/serialize if using virtual file name fields
         {
-            for (unsigned s=0; s<numsubs; s++)
+            IDistributedSuperFile *super = file->querySuperFile();
+            if (super)
+            {
+                unsigned numsubs = super->numSubFiles(true);
+                unsigned s=0;
+                for (; s<numsubs; s++)
```

Comment:
trivial/style: normally counter var. ('s') declared inside for

Reply:
My intention was to undo the changes made in the PR and this was how it was before but I'll make the improvement.